### ISSUE TITLE
Overzoom detection: try first lower zoom level instead not doing anything

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#457]
Right now when overzooming detects that the chosen GUI size won't fit your screen, I feel it would be better to try the next lower zoom level, then recurse that if it still doesn't fit (say you selected 300% on 1080p monitor). This way we could end up with the best possible fit, instead of nothing happening and requiring the user to visit the menu again to choose another zoom level.

Thanks!

**Comment by baconpaul** [comment]
Yeah for the @sense-amr issue #456 I need a best-fit-for-monitor heuristic also so I can do both at the same time. Thanks!

Probably won't do these this weekend but we can get to em. Good ideas both.

**Comment by baconpaul** [comment]
Oh and @sense-amr thanks for tagging me! But I get notifications for all issues already since I'm one of the maintainers. Appreciate it but I would have seen this even if you hadn't. No worries that you did of course!

**Comment by sense-amr** [comment]
ahh yeah i just remember it was something we discussed.. but i didnt know it was possible on AU .. im not developing on the mac much or using it .. mostly PC testing here. .
vst2 and vst3

**Comment by baconpaul** [comment]
<img width="509" alt="screen shot 2019-02-04 at 9 14 00 am" src="https://user-images.githubusercontent.com/13387561/52213503-479c3580-285d-11e9-90b7-7c25afae14a7.png">

OK here's the new message. All part of my big zoom cleanup branch. Should be headed to a PR today sometime. Maybe tomorrow latest.

**Comment by baconpaul** [comment]
I could make you one which goes up to 98%. Then you would go to that. Then you would ask me to back it down. Because you would have lost the menu button because of the menu bar and window decoration.

Remember: This is setting the size of the surge window INSIDE the decorated window sitting in a desktop with other components. On a mac you loose 20 pixels or so for the menu bar at the top. You lose another chunk for the window border. On windows it is worse.

**Comment by baconpaul** [comment]
The easiest fix is to just lie to the user and not show them the 95% thing at all! I'll do that before I do the PR.
